src/planwise/client/projects2/components/dashboard.cljs

```diff
@@ -8,11 +8,19 @@
             [planwise.client.routes :as routes]
             [planwise.client.ui.common :as ui]
             [planwise.client.ui.rmwc :as m]
-            [planwise.client.utils :as utils]))
+            [planwise.client.utils :as utils]
+            [planwise.client.projects2.components.settings :as settings]))
 
 (defn- project-tabs
-  [{:keys [active] :or {active 0}}]
-  [m/TabBar {:activeTabIndex active}
+  [{:keys [active] :or {active :scenarios}}]
+  [m/TabBar {:activeTabIndex (cond
+                               (= active :scenarios) 0
+                               (= active :settings) 1)
+             :on-change (fn [evt]
+                          (let [tab-index (.-value (.-target evt))]
+                            (cond
```

Comment:
Ditto for `case` vs. `cond`. Or `dispatch` on the mapping of `tab-index` into the handler names (careful with not dispatching `nil` though).